logging-k8s-stdout-otlp-json/k8s/collector-configmap.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: otel-collector-config
+data:
+  otel-collector-config.yaml: |-
+    receivers:
+      otlp:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:4317
+          http:
+            endpoint: 0.0.0.0:4318
+      prometheus/collector: # needed if you use the docker-lgtm image
```

Comment:
do we need this `prometheus` receiver for our example?

Reply:
yes, for the automated tests